config/new.go

```diff
@@ -96,6 +98,10 @@ func NewHCPConfig(opts ...HCPConfigOption) (HCPConfig, error) {
 		}
 	}
 
+	if config.suppressLogging {
+		log.SetOutput(ioutil.Discard)
```

Comment:
```suggestion
		log.SetOutput(io.Discard)
```

`ioutil` is deprecated in favor of `io`